test/test_unit_cli.py

```diff
@@ -396,10 +399,12 @@ def test_run_action_runs_init_if_conjurrc_not_found(
     @patch('conjur.data_object.conjurrc_data.ConjurrcData.load_from_file',
            return_value=MockConjurrc)
     @patch('keyring.get_keyring')
+    @patch('conjur.util.util_functions.get_netrc_flag_from_conjurrc')
```

Comment:
Did some digging and discovered why it initially only with a mocked version of `get_netrc_flag_from_conjurrc`. It's because this way avoided making additional call to `os.path.exists` added by the new util function. `os.path.exists` is also mocked with the side_effect directive specifying the first two return values. Adding a third l call via `get_netrc_flag_from_conjurrc` was throwing off the expected return value. 

I was able to address it by simply adding an additional boolean value to the side_effect array. Thanks for pointing it out!